guides/user-manual/json_rpc.rst

```diff
@@ -111,7 +111,7 @@ To use ``mmWSClient`` in a PowerShell session:
 
       > .\mmWSClient.ps1 -mmWSUri http://central-web.demo/mmws -centralServer central.demo -username a.user -password secret
 
-These arguments are optional, as the API assumes the Micetro Central instance is located on the same machine as the Web Services by default. The API also supports other authentication mechanisms like Basic Authentication, Windows NTLM, and Kerberos.
+These arguments are optional, as the API assumes the Micetro Central instance is located on the same machine as the Web Services by default. The API also supports other authentication mechanisms like Bearer Authentication, Windows NTLM, and Kerberos.
```

Comment:
The "The API also supports..." sentence is less relevant here. The new version of mmWSClient just uses Bearer auth (as opposed to setting the session in the request body), since it's the preferred way for API usage, but I don't think we need to mention that here